roles/slurm/tasks/controller.yml

```diff
@@ -1,20 +1,45 @@
 ---
 
-- name: install mariadb
+- name: install dependencies
   apt:
     name: "{{ item }}"
     state: latest
   with_items:
     - mariadb-server
     - python-mysqldb
+    - s-nail
+    - ssmtp
+  when: ansible_os_family == "Debian"
 
-- name: install s-nail and ssmtp
-  apt:
+- name: install dependencies
+  yum:
     name: "{{ item }}"
     state: latest
   with_items:
-    - s-nail
+    - mariadb-server
+    - MySQL-python
+    - mailx
     - ssmtp
+  when: ansible_os_family == "RedHat"
+
+- name: edit mariadb selinux policy
+  shell: grep mysqld /var/log/audit/audit.log | audit2allow -M mariadb_local
+  args:
+    creates: mariadb_local.pp
+  register: policy_created
```

Comment:
I think this var gets registered even if `audit2allow` doesn't have anything to do, it has rc 0.